app/controllers/DepartureMessagesController.scala

```diff
@@ -56,6 +56,8 @@ trait V1DepartureMessagesController {
 
   def getDepartureMessages(departureId: DepartureId, receivedSince: Option[OffsetDateTime]): Action[AnyContent]
 
+  def sendMessageDownstream(departureId: DepartureId): Action[NodeSeq]
```

Comment:
you might want to rename this to attachMessage as well

Reply:
This is a v1 method, we want to change this as little as possible - this should be left as is.